build-system/pr-check/utils.js

```diff
@@ -135,7 +135,7 @@ function startTimer(jobNameOrCmd) {
  * Stops the timer for the given job / command and prints the execution time.
  * @param {string} jobNameOrCmd
  * @param {DOMHighResTimeStamp} startTime
- * @return {number}
+ * @return {void}
```

Comment:
We should just remove the `@return` annotation here and everywhere else that's now a `void`, right?

Reply:
That should be fine.